lib/rspec/buildkite/analytics/reporter.rb

```diff
@@ -18,6 +18,7 @@ def handle_example(notification)
 
       if trace
         trace.example = example
+        populate_failure_info(trace, notification)
```

Comment:
I think we should move the mutation of the trace out of the `populate_failure_info` method. It could return an array of `[reason, expanded]` and then destructure like so.
```suggestion
        trace.failure_reason, trace.failure_expanded = failure_info(notification)
```